corehq/util/celery_utils.py

```diff
@@ -127,13 +127,21 @@ def revoke_tasks(task_names, interval=5):
 
         for current_task in tasks:
             if current_task.name in task_names and current_task.id not in task_ids:
-                app.control.revoke(current_task.id, terminate=True)
+                revoke_task(current_task.id, celery_app=app)
                 task_ids.add(current_task.id)
                 print(datetime.utcnow(), 'Revoked', current_task.id, current_task.name)
 
         sleep(interval)
 
 
+def revoke_task(task_id, celery_app=None):
+    if celery_app is None:
+        app = Celery()
+        app.config_from_object('django.conf:settings', namespace='CELERY')
```

Comment:
I assume you need to set `celery_app` to `app` in this block as well?

Reply:
sorry yes